content/applications/inventory_and_mrp/manufacturing/workflows/continuous_improvement.rst

```diff
@@ -0,0 +1,233 @@
+==============================
+Continuous product improvement
+==============================
+
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+.. |MOs| replace:: :abbr:`MOs (Manufacturing Orders)`
+.. |BoM| replace:: :abbr:`BoM (Bill of Materials)`
+.. |ECO| replace:: :abbr:`ECO (Engineering Change Order)`
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+.. |PLM| replace:: :abbr:`PLM (Product Lifecycle Management)`
+.. |BoMs| replace:: :abbr:`BoMs (Bills of Materials)`
+
+*Continuous improvement* is a general philosophy intended to help individuals and organizations
+constantly improve themselves and the work they produce.
+
+There are a variety of different methodologies that fall under the umbrella of continuous
+improvement. These include kaizen, six sigma, and lean, among others. While the specific steps of
+each method differ, their goal remains the same: implement a process by which improvement is a
+perpetual goal, rather than a one-time accomplishment.
+
+The sections below contain details about how Odoo can be used to implement four general steps common
+to many of the most popular continuous improvement strategies, with links to documentation about
+configuring the necessary features. The final section details how a specific company might configure
+these Odoo implementations within their organization.
+
```

Comment:
Suggestion to add references of each section :)
(will need to add custom anchors) 
```suggestion

#. :ref:`manufacturing/workflows/ci-identify`
#. :ref:`manufacturing/workflows/ci-suggest`
#. :ref:`manufacturing/workflows/ci-implement`
#. :ref:`manufacturing/workflows/ci-review`
```